app/bingey-api/server.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const mongoose = require('mongoose');
 const bodyParser = require('body-parser');
```

Comment:
As `bodyparser` package is deprecated and `app.use(express.json())` has its features, we could remove the dependency.
Also, grouping the imports into external and internal ones makes sense to me. Same for the `ap.use(...)` - grouping middleware separately from our routes makes for easier maintenance